tests/sentry/api/endpoints/test_project_rules.py

```diff
@@ -84,14 +79,7 @@ def test_with_environment(self):
 
         Environment.get_or_create(project, "production")
 
-        conditions = [
-            {
-                "id": "sentry.rules.conditions.first_seen_event.FirstSeenEventCondition",
-                "key": "foo",
```

Comment:
not sure what these extra keys (`key`, `match`, `value`) were for, but I deleted them because it seems like `FirstSeenEventCondition` doesn't require it